reference/configuration/framework.rst

```diff
@@ -204,21 +204,20 @@ ide
 
 **type**: ``string`` **default**: ``null``
 
-If you're using an IDE like TextMate or Mac Vim, then Symfony can turn all
-of the file paths in an exception message into a link, which will open that
-file in your IDE.
+Symfony can turn file paths seen in dumps and in exception messages into links
```

Comment:
you can remove the "in" before "exception messages", which will improve the read flow a bit